pkg/cli/cmds/agent.go

```diff
@@ -8,36 +8,33 @@ import (
 
 var (
 	k3sAgentBase = mustCmdFromK3S(cmds.NewAgentCommand(AgentRun), map[string]*K3SFlagOption{
-		"v":               Hide,
-		"vmodule":         Hide,
-		"log":             Hide,
-		"alsologtostderr": Hide,
-		"data-dir": {
-			Usage:   "(data) Folder to hold state",
-			Default: "/var/lib/rancher/rke2",
-		},
-		"token":                      nil,
-		"token-file":                 nil,
-		"disable-selinux":            Drop,
-		"node-name":                  nil,
-		"with-node-id":               Drop,
-		"node-label":                 nil,
-		"node-taint":                 nil,
-		"docker":                     nil,
-		"container-runtime-endpoint": nil,
-		"pause-image":                Drop,
-		"private-registry":           nil,
-		"node-ip":                    nil,
-		"node-external-ip":           Drop,
-		"resolv-conf":                nil,
-		"flannel-iface":              Drop,
-		"flannel-conf":               Drop,
-		"kubelet-arg":                nil,
-		"kube-proxy-arg":             nil,
-		"rootless":                   Drop,
-		"server":                     nil,
-		"no-flannel":                 Drop,
-		"cluster-secret":             Drop,
+		"v":                          hide,
+		"vmodule":                    hide,
+		"log":                        hide,
+		"alsologtostderr":            hide,
+		"data-dir":                   copy,
```

Comment:
shouldnt this be overriden to /var/lib/rancher/rke2 instead of /var/lib/rancher/k3s

Reply:
I think this qualifies per @ibuildthecloud 's comment from above about this commit: https://github.com/rancher/k3s/commit/7e59c0801e905f9f574264326ba42143c665b3b5